perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -549,6 +549,16 @@ sub is_open {
     return exists $self->open_states->{ $self->state } ? 1 : 0;
 }
 
+=head2 is_in_progress
+
+Sees if the problem is in an open, not 'confirmed' state.
+
+=cut
+
+sub is_in_progress {
```

Comment:
I know it's only a small method, but it'd be good if we could have some tests for this 😄